docs/source/json-api/index.rst

```diff
@@ -20,7 +20,6 @@ complicating concerns including, but not limited to:
 - inspecting transactions,
 - asynchronous submit/completion workflows,
 - temporal queries (e.g. active contracts *as of a certain time*), and
-- ledger metaprogramming (e.g. retrieving packages and templates).
```

Comment:
slightly confusing as not completely true anymore, JSON API provides some pkg management